src/desktop/apps/artwork/client.tsx

```diff
@@ -25,6 +27,8 @@ buildClientApp({
 })
   .then(({ ClientApp }) => {
     ReactDOM.hydrate(<ClientApp />, document.getElementById("react-root"))
+    // TODO: Remove after inquiry a/b test
+    splitTest("inquiry_auth").view()
```

Comment:
Do we want to only fire this when there is a user? I can't 100% tell if the test itself only 'makes sense' for logged in users (if so, the `view()` should only fire when logged in).

Reply:
Good question, I'll check with the data team.

Reply:
Either way I think it makes sense to move this to when we trigger opening the inquiry rather than for all artist page viewers.